build.gradle.kts

```diff
@@ -81,8 +81,12 @@ kotlin {
 }
 
 tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach {
-   kotlinOptions.jvmTarget = "1.8"
-   kotlinOptions.apiVersion = "1.4"
+   kotlinOptions {
+      freeCompilerArgs = freeCompilerArgs + listOf("-Xopt-in=kotlin.RequiresOptIn", "-Xopt-in=kotlin.time.ExperimentalTime")
```

Comment:
I wonder if this doesn't work because of something related to `buildSrc` 🤔

Reply:
Does it not compile or ... ?

Reply:
You can just list on multiple lines too

freeCompilerArgs = freeCompilerArgs + "-Xopt-in=kotlin.RequiresOptIn"
freeCompilerArgs = freeCompilerArgs + "-Xopt-in=kotlin.time.ExperimentalTime"

Reply:
It doesn't get rid of the warning about `RequiresOptin` and doesn't auto optin to `ExperimentalTime` unfortunately

Reply:
I can't get it to work on MPP either